Makefile

```diff
@@ -42,22 +41,85 @@ godoc: ## runs godoc server and opens in browser
 	@# The read varname command will keep the make command running (waiting for input) until you press CTRL+C.
 	@trap 'kill `cat godoc.pid` && rm -f godoc.pid' EXIT; read varname
 
+.PHONY: fmt
 fmt: ## formats all go files
 	go fmt ./...
 	make format-lines
 
+.PHONY: format-lines
 format-lines: ## formats all go files with golines
 	go install github.com/segmentio/golines@latest
 	golines -w -m 120 --ignore-generated --shorten-comments --ignored-dirs=${GO_LINES_IGNORED_DIRS} ${GO_FOLDERS}
 
+.PHONY: lint
 lint: ## runs all linters
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
 	golangci-lint run ./...
 
+
+___BINDINGS___: ## 
+
+core_default := "DelegationManager IRewardsCoordinator ISlasher StrategyManager EigenPod EigenPodManager IStrategy IAVSDirectory"
+core_location := "./lib/eigenlayer-middleware/lib/eigenlayer-contracts"
+core_bindings_location := "../../../../bindings"
+
+middleware_default := "RegistryCoordinator IndexRegistry OperatorStateRetriever StakeRegistry BLSApkRegistry IBLSSignatureChecker ServiceManagerBase IERC20"
+middleware_location := "./lib/eigenlayer-middleware"
+middleware_bindings_location := "../../bindings"
+
+sdk_default := "MockAvsServiceManager ContractsRegistry"
+sdk_location := "."
+sdk_bindings_location := "./bindings"
+
+.PHONY: core-bindings ## generates core contracts bindings
+core-bindings: ## generates core bindings
+	@echo "Starting core bindings generation"
+ifneq ($(contracts),)
```

Comment:
What is this contracts variable and where does it come from?

Reply:
so if you just want to generate binding of a single contract you can pass like `make core-bindings contracts=DelegationManger" it will just generate that binding.

Reply:
fa01f4851fe77da845925b317787eae58f92de22